third_party/move/move-compiler-v2/src/inliner.rs

```diff
@@ -64,6 +65,7 @@ type CallSiteLocations = BTreeMap<(QualifiedFunId, QualifiedFunId), BTreeSet<Nod
 /// Run inlining on current program's AST.  For each function which is target of the compilation,
 /// visit that function body and inline any calls to functions marked as "inline".
 pub fn run_inlining(env: &mut GlobalEnv) {
+    info!("Inlining");
```

Comment:
It might be nice to have some guidelines as to what would qualify to be `info!()`-worthy messages.

Reply:
This would be also part of the discussion.